crates/runtime/src/mgmt.rs

```diff
@@ -960,8 +959,8 @@ fn command_insert_into_gen_block(
                     lc_dict_data: None,
                 },
             });
-        }
         ic += 1;
```

Comment:
the moving cause the format problem, which I have fixed in another branch [fix_more_warnings](https://github.com/tensorbase/tensorbase/tree/fix_more_warnings). So no problem. accepted.